imap_processing/cdf/config/imap_hi_variable_attrs.yaml

```diff
@@ -0,0 +1,271 @@
+# ------- Default attributes section -------
+default_attrs: &default
+  DEPEND_0: epoch
+  DISPLAY_TYPE: time_series
+  FORMAT: I12
+  UNITS: " "
+  VAR_TYPE: data
+  SCALE_TYPE: linear
+
+default_uint8_attrs: &default_uint8
```

Comment:
cool. I really like these some other defaults option. Would love to do same for mine. In the future, this would be a nice shared defaults. Also I like that you are adding `dtype`. Would like to do same in the future.

Reply:
Yes! I agree that figuring out a way to add attribute inheritance across yaml files would be beneficial and allow everyone to use these type specific defaults.